src/components/DayOverview/DayOverviewPdf.tsx

```diff
@@ -0,0 +1,185 @@
+"use client";
+
+import React from "react";
+import { Svg, Document, Page, Text, View, StyleSheet, Path } from "@react-pdf/renderer";
+import { faFlag, faSquare, IconDefinition } from "@fortawesome/free-solid-svg-icons";
+
+import {
+    DayOverviewData,
+    getCurrentDate,
+    ParcelOfSpecificDateAndLocation,
+} from "@/components/DayOverview/DayOverview";
+
+interface DayOverviewSubtitleProps {
+    subtitleKey: string;
+    children: string;
+}
+
+interface DayOverviewRowProps {
+    row: ParcelOfSpecificDateAndLocation;
+}
+
+interface DayOverviewContentProps {
+    date: Date;
+    location: string;
+    data: ParcelOfSpecificDateAndLocation[];
+}
+
+interface DayOverviewPdfProps {
+    data: DayOverviewData;
+}
+
+interface CustomSVGProps {
+    icon: IconDefinition;
+    color: string;
+    fill: boolean;
+}
+
+const styles = StyleSheet.create({
+    page: {
+        display: "flex",
+        flexDirection: "column",
+        padding: "0 30pt 15pt",
+        fontSize: "11pt",
+    },
+    title: {
+        fontFamily: "Helvetica-Bold",
+        textTransform: "uppercase",
+        fontSize: "30px",
+        textAlign: "center",
+        margin: "0 22.5pt 22.5pt",
+    },
+    subtitleWrap: {
+        justifyContent: "space-between",
+        marginBottom: "4pt",
+    },
+    svg: {
+        margin: "0 0.75pt",
+        padding: "0 0.75pt",
+    },
+    cell: {
+        borderStyle: "solid",
+        borderRightWidth: 1,
+        borderBottomWidth: 1,
+        padding: "2.25pt",
+    },
+    margin: { height: "37.5pt" },
+    row: { display: "flex", flexDirection: "row" },
+    bold: { fontFamily: "Helvetica-Bold" },
```

Comment:
If this is all that is in styles.bold it can be added inline (dw tho if you think this is nicer)

Reply:
Feels like styles.bold is nicer because if e.g, future developer want to change fonts we can just change one fontFamily and not change all the inline code